diff --git a/Ghidra/Features/Decompiler/src/decompile/cpp/heritage.cc b/Ghidra/Features/Decompiler/src/decompile/cpp/heritage.cc
index 0f73153..d932c7d 100644
--- a/Ghidra/Features/Decompiler/src/decompile/cpp/heritage.cc
+++ b/Ghidra/Features/Decompiler/src/decompile/cpp/heritage.cc
@@ -1038,6 +1038,7 @@ bool Heritage::discoverIndexedStackPointers(AddrSpace *spc,vector<PcodeOp *> &fr
 	  if (op->getIn(2) != curNode.vn) break;	// Check that stackpointer comes in as inner pointer
 	  // Treat output as having the same offset, fallthru to COPY
 	}
+	case CPUI_INT_AND:	// Stack alignment - treat as copy (offset unchanged)
 	case CPUI_INDIRECT:
 	case CPUI_COPY:
 	{
